lib/puppet/type/registry_key.rb

```diff
@@ -128,4 +128,6 @@ def eval_generate
     end
     resources
   end
+  # rubocop:enable Metrics/MethodLength
```

Comment:
could probably remove `# rubocop:enable Metrics/MethodLength ` here as its duplicated in L133

Reply:
again, methodlength vs blocklength